app/services/builders/chartkick/department_data.rb

```diff
@@ -0,0 +1,12 @@
+module Builders
+  module Chartkick
+    class DepartmentData < Builders::Chartkick::Base
```

Comment:
I would rename the class to `DepartmentDataRetriever` or to `DepartmentDataService` to make it clear that the object is not the DepartmentData itself but it returns DepartmentData

Reply:
In fact is a data builder, I believe that `Builders::Chartkick::DepartmentData` is clear enough. WDYT @hvilloria ?

Reply:
Maybe you are right. I read only the class name but in its uses it will have the fully qualified name